src/Repl.vue

```diff
@@ -63,14 +66,18 @@ if (!props.editor) {
   throw new Error('The "editor" prop is now required.')
 }
 
-const outputRef = useTemplateRef('output')
+const outputRef = useTemplateRef<typeof Output>('output')
```

Comment:
Use the latest Volar, the generic type will be inferred automatically.

Reply:
Thanks for the reminder, it has been updated.